docs/development/setup.md

```diff
@@ -0,0 +1,25 @@
+# Setup
+
+Ensure you have the following installed first:
+
+- librocksdb-dev
+- libclang-dev
+- clang lldb lld
+- build-essential
+- protoc
+
```

Comment:
Please add commands for installing each package

Reply:
It's different for every OS, so will not add that for now